app/models/regular_transaction.rb

```diff
@@ -1,5 +1,5 @@
 class RegularTransaction < ApplicationRecord
-  FREQUENCIES = %w[weekly monthly].freeze
+  FREQUENCIES = %w[every_week every_two_weeks every_four_weeks monthly total_in_last_three_months].freeze
```

Comment:
be good too use the expected values in CFE so they can be persisted and passed across to cfe as is. You can see the [current values here](https://github.com/ministryofjustice/check-financial-eligibility/blob/main/app/constants/cfe_constants.rb#L72)

Reply:
Have updated it to use these values in the backend and render the right content in the frontend :)